### PR TITLE
Add conda installation instructions (fixes #7)

### LINKED ISSUE #7: add conda installation instructions
With MDAnalysis available through `conda` (MDAnalysis/mdanalysis#608) and the blog post announcement imminent (MDAnalysis/MDAnalysis.github.io#20), we should also update the tutorial to provide the conda-based installation as a simple starting point.

### PR DESCRIPTION
**Pull request by kain88-de**
fixes #7

### CODE REVIEW

**Comment by kain88-de** [comment]
@orbeckst I'm assigning you since you have been so friendly to assign me to the bug :wink:

File: doc/sphinx/installation.rst [hunk 1]
```diff
@@ -54,8 +54,6 @@ If there are problems then please have a closer look at the
 If you need help with installation issues, please do not hesitate to
 ask on the `user discussion group`_.
 
-For this tutorial you can also alternatively use a complete
```
Comment (orbeckst) [inline doc/sphinx/installation.rst]: Good idea to rather try conda instead of VM! +1

File: doc/sphinx/installation.rst [hunk 2]
```diff
@@ -65,15 +63,34 @@ installation inside a :ref:`virtual-machine`.
...
+
+The latest release of MDAnalysis_ (and the full test suite) can be
+installed from the `Anaconda.org channel`_ with conda_ ::
+
+  conda config --add channels MDAnalysis
+  conda install mdanalysis
+
+The conda package includes the testsuite by default.
+
+.. _conda: https://anaconda.org/MDAnalysis
```
Comment (orbeckst) [inline doc/sphinx/installation.rst]: should be link to http://conda.pydata.org/ (or some other tutorial-style docs that gets people up and running fast. I would actually recommend to [download and install the Anaconda Installer](https://www.continuum.io/downloads).

Comment (kain88-de) [inline doc/sphinx/installation.rst]: that is a mistake. It should point to one the pages you named. 

Btw I have gotten almost  of my colleagues to use conda with the new MDAnalysis package. So far everything is fine. People are very happy about the painless installation

**Comment by kain88-de** [comment]
@orbeckst I've addressed your comment

**Pull request merged**